qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
```

Comment:
not good to list compare mode here. better way is  compare mode in variants of configuration.